src/NuGet.Clients/NuGet.VisualStudio/IVsPackageManagerProvider.cs

```diff
@@ -1,48 +0,0 @@
-using System.Runtime.InteropServices;
-using System.Threading;
-using System.Threading.Tasks;
-
-namespace NuGet.VisualStudio
-{
-    /// <summary>
-    /// Interface allowing integration of alternate package manager suggestion for a NuGet package. 
-    /// For example jQuery may appear on Bower and npm,
-    /// it might be more appropriate to install a package from them for certain projects. 
-    /// </summary>
-    [ComImport]
-    [Guid("BCED5BF2-40FC-4D9F-BF0A-43CD4E9FF65F")]
-    public interface IVsPackageManagerProvider
```

Comment:
This is a breaking change. How are we going to communicat to other teams about this change?

Reply:
The only team I can find using this is ASP.NET for Bower, and they agreed to this removal. I'm going going to file a bug against them to remove their provider.

Reply:
I think our approach for this should likely be us *not importing* any providers, rather than removing them from the interop assembly altogether.

Reply:
Gotcha. Are you saying restore this file, or is there more to it than that?

Reply:
Yep, restore this file and changes to the public shipped API file, and I think the rest of the changes stand.